src/pydocstyle/checker.py

```diff
@@ -764,6 +764,9 @@ def _check_missing_args(docstring_args, definition):
             # positional argument as it is `cls` or `self`
             if definition.kind == 'method' and not definition.is_static:
                 function_args = function_args[1:]
+            # Filtering out any arguments prefixed with `_` marking them
+            # as private.
+            function_args = [arg for arg in function_args if not arg.startswith("_")]
```

Comment:
Can you extract this logic to a function? e.g., `is_def_arg_private`. The code will have clearer intent (`if not is_def_arg_private(arg)`) and can be reused.